templates/web-application/README.md

```diff
@@ -0,0 +1,64 @@
+# {{PACKAGE-NAME}}
+
+> Your web-application information go here
+
+## Scripts
+
+### dev
+
+Start a development server at the address [http://localhost:8080](http://localhost:8080).
+
+```bash
+pnpm dev
+```
+
+### build
+
+Build {{PACKAGE-NAME}} as production ready and output the result in `dist`;
+
+```bash
+pnpm build
+```
+
+### test
+
+Run the tests on the sections of {{PACKAGE-NAME}}.
+
+```bash
+pnpm test
+```
+
+### serve-build
+
+Run a local server on the content of the `dist` folder at the address [http://localhost:8080](http://localhost:8080).
+
+```bash
+pnpm serve-build
+```
+
+### reset
+
+Delete all build results and `pnpm` installed packages in this project.
+
+```bash
+pnpm reset
+```
+
+Run linting on {{PACKAGE-NAME}} files.
+
+### lint
+
+```bash
+pnpm lint
+```
+
+### analyze
+
+Run Webpack with bundle analyzer on the result of a production-ready build.
```

Comment:
I found that blog article that explain how to use it: https://blog.jakoblind.no/webpack-bundle-analyzer/#what-should-i-look-for-in-the-reports

I can also write something to explain how to use the report.

Reply:
For example, by using ChatGPT:

# How to analyze Webpack reports:

1. *Understand the Report*: Webpack bundle analyzer provides a visual report of your blog. Each generated JS file is represented by a colored box, the size of which corresponds to the size of the JS file. The larger boxes are often for framework files or dependencies, and your own written code is usually in smaller boxes. You can hover over a box to see the exact file size and gzip size, and you can zoom in or out to see details.
2. *Identify Large Libraries*: If you notice large libraries, check if they are tree-shakable, which means you only bundle the part of the library you use. For example, lodash is often a large library that can be tree-shaken. If lodash appears in your bundle but not in your package.json, it might be a dependency of another package. In this case, you can check package-lock.json to identify the dependent package.
3. *Check for Deprecated Libraries*: Look for deprecated libraries like Moment.js. The best way to reduce your bundle size is to remove such libraries completely. Modern JavaScript often allows for the functionality of these deprecated libraries without the need for additional packages.
4. *Analyze Large Bundles*: Webpack suggests that a bundle should be a maximum of 250kb unminified. If any bundles exceed this size, consider using code splitting to break them up and potentially lazy load parts of them.
5. *Look for Duplications*: If you notice duplicated code across many bundles, consider splitting that code into its own bundle to reduce repetition and increase efficiency.

By following these steps, you can better understand and optimize your Webpack bundle, ultimately improving the performance of your blog or website.

Reply:
https://blog.jakoblind.no/webpack-bundle-analyzer/#what-should-i-look-for-in-the-reports is great! Let's add it :)